app/handlers/web/auth.go

```diff
@@ -93,8 +93,9 @@ func AuthGithubCallbackHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if auth.IsFirstUser(db) {
-		auth.CreateNewGithubUser(db, user, token.AccessToken)
-		http.Redirect(w, r, "/", http.StatusTemporaryRedirect)
+		user,_ := auth.CreateNewGithubUser(db, user, token.AccessToken)
+		token := auth.CreateToken(&user)
```

Comment:
Isn't `token` is already initialized [here](https://github.com/generationtux/brizo/blob/db92cedddefd9ea2a1f7adcff991335086a80ce5/app/handlers/web/auth.go#L72)? We may want to consider more specific names like `jwtToken` and `oauthToken` either way.